bloom/commands/git/import_upstream.py

```diff
@@ -39,7 +39,7 @@
 import tarfile
 import tempfile
 
-from pkg_resources import parse_version
+from packaging.version import parse
```

Comment:
Being a bit of a python outsider, I've always disliked that a function's utility is sometimes obfuscated when the functions are imported outside their module.
Does using the alias-on-import feature make sense so that `parse` doesn't lose its context.
If using the alias-on-import feature is not seen as idiomatic then I could live with it as is, but I think it improves local legibility.

```suggestion
from packaging.version import parse as parse_version
```

Reply:
Another alternative that is used in the other files is to import the entire version module which would afford the same context as the function would be invoked as `version.parse()`.

```suggestion
from packaging import version
```

Reply:
I would say the function name should just be something more specific than parse, but I think your first suggestion is just fine for now.

Reply:
It looks like I've avoided using ``from packaging import version``, because the parameter for the ``version_check`` method was also ``version`` and there was a conflict.

But I can go with parse_version.